### PR TITLE
Conclusion Page Hifi

### PR DESCRIPTION
**Pull request by jaydhulia**
https://www.loom.com/share/4b179c82a2614871bac245e593476877

### CODE REVIEW

File: app/public/static/css/completed_page.css [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  padding-top: 3.90625vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.back-to-lobby-button {
+  /*width: 133px;*/
+  /*height: 44px;*/
+  /*padding-top: 3.90625vh;*/
```
Comment (ahmedhamodi) [inline app/public/static/css/completed_page.css:149]: Let's remove, unless it may be needed in the future.

**Review by ahmedhamodi — APPROVED** [review]
Code approval - looks good to me.

**Comment by jaydhulia** [comment]
Cleaned up the code, added padding and added "Link Copied" button on click that changes back to "Copy Link" after 3 seconds. Updated loom video: https://www.loom.com/share/21fa470f082c4ae8a0552af7ece51307

**Review by ahmedhamodi — APPROVED** [review]
lgtm

**Comment by jaydhulia** [comment]
> Could we perhaps remove the 3 second change back and keep it like "link copied" unless the other link copy is clicked?

Oops, accidentally closed. Updated loom video: https://www.loom.com/share/364a0218369d47f9a65319349a64e0e8

**Review by aaronabraham311 — APPROVED** [review]
Product approval

**Pull request merged**